set.mm

```diff
@@ -262804,6 +262880,28 @@ the ring of the coefficients is not a division ring (not a field in the
       VIULUPVJVOACWGJACVTVETZWGACKVETWKMAKVTVEWDUPVJCWGVTVRWEWJVKVLQVMVNVP $.
   $}
 
+  ${
+    frlmvplusgvalc.f $e |- F = ( R freeLMod I ) $.
+    frlmvplusgvalc.b $e |- B = ( Base ` F ) $.
+    frlmvplusgvalc.r $e |- ( ph -> R e. V ) $.
+    frlmvplusgvalc.i $e |- ( ph -> I e. W ) $.
+    frlmvplusgvalc.x $e |- ( ph -> X e. B ) $.
+    frlmvplusgvalc.y $e |- ( ph -> Y e. B ) $.
+    frlmvplusgvalc.j $e |- ( ph -> J e. I ) $.
+    frlmvplusgvalc.a $e |- .+ = ( +g ` R ) $.
+    frlmvplusgvalc.p $e |- .+b = ( +g ` F ) $.
+    $( Addition in a free module at a coordinate.  (Contributed by AV,
+       16-Jan-2023.) $)
```

Comment:
Coordinates of a sum with respect to a basis in a module.

Reply:
To explain my above proposal: it mentions a basis, which the original comment fails to do, and "at a coordinate" has little sense.

Side remark: freeness, for a module, is a property, not the data of a particular basis.

Reply:
frlmvscaval: Coordinates of a scalar multiple with respect to a basis in a module.

Maybe not perfect, but "at a coordinate" just makes no sense.

Reply:
Comments revised according to BJ's proposals.